test/jdk/java/text/Format/MessageFormat/CompactSubFormats.java

```diff
@@ -23,10 +23,10 @@
 
 /*
  * @test
- * @bug 8318761
+ * @bug 8318761 8329118
  * @summary Test MessageFormatPattern ability to recognize and produce
  *          appropriate FormatType and FormatStyle for CompactNumberFormat.
- * @run junit CompactSubFormats
+ * @run junit/othervm -Duser.language=en -Duser.country=US CompactSubFormats
```

Comment:
I would instantiate MessageFormat explicitly with the `US` locale (using the 2-arg constructor), instead of implicitly specifying it with the system property.

Reply:
Thanks for the review and suggestions, PR should be updated. (I presume you meant the 2-arg Locale.of() method, not the constructor)

Reply:
I meant using `new MessageFormat(String, Locale.US)`, instead of `new MessageFormat(String)`.

Reply:
Ah, gotchu. Updated with the two arg `MessageFormat` constructor, although I think it would have been fine previously as well just passing the locale in directly to the subformat.